habitat/tasks/rearrange/sub_tasks/pick_task.py

```diff
@@ -124,16 +124,16 @@ def _gen_start_pos(self, sim, is_easy_init):
     def _should_prevent_grip(self, action_args):
         return (
             self._sim.grasp_mgr.is_grasped
-            and action_args.get("grip_ac", None) is not None
-            and action_args["grip_ac"] <= 0
+            and action_args.get("grip_action", None) is not None
```

Comment:
I agree we should refactor this hard coded string to be defined somewhere since it is used many places throughout the code.